src/main/java/org/apache/commons/net/nntp/NewGroupsOrNewsQuery.java

```diff
@@ -246,7 +246,7 @@ public boolean isGMT() {
      * </pre>
      * <p>
      *
-     * @param newsgroup The newsgroup to add to the list of groups to be checked for new news, but which should be omitted from the search for new news..
+     * @param newsgroup The newsgroup to add to the list of groups to be checked for new news, but which should be omitted from the search for recent news.
```

Comment:
Why "fresh" before and "recent" here? Using "recent" for both would be more consistent IMO.